api/app/controllers/spree/api/v1/products_controller.rb

```diff
@@ -98,6 +96,10 @@ def product_params
             params.require(:product).permit(permitted_product_attributes)
           end
 
+          def find_product
```

Comment:
@tanmay3011 private/protected methods shouldn't be indented more than any other methods, let's keep to rubocop's rules :)